### PR TITLE
fix(TUP-34954)CVE: Replace log4j1.x by reload4j

### PR DESCRIPTION
**Pull request by jiezhang-tlnd**

### CODE REVIEW

**Review by bhe-talendbj — APPROVED** [review]

File: main/plugins/org.talend.commons.runtime/pom_server.xml [hunk 1]
```diff
@@ -95,11 +95,11 @@
...
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.16</version>
-            <type>bundle</type>
-        </dependency>
+			<groupId>ch.qos.reload4j</groupId>
+			<artifactId>reload4j</artifactId>
+			<version>1.2.19</version>
+			<type>bundle</type>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.commons.runtime/pom_server.xml:101]: remove `<type>bundle</type>`
1 - reload4j is already a bundle
2 - there is no type bundle for reload4j in our nexus.

File: main/plugins/org.talend.utils/pom_minimal.xml [hunk 2]
```diff
@@ -38,9 +38,9 @@
             <version>1.9.12</version>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.16</version>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>1.2.19</version>
             <type>bundle</type>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.utils/pom_minimal.xml:44]: Same as above

File: main/plugins/org.talend.utils/pom_server.xml [hunk 3]
```diff
@@ -38,9 +38,9 @@
             <version>1.9.12</version>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.16</version>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>1.2.19</version>
             <type>bundle</type>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.utils/pom_server.xml:44]: Same as above

File: main/plugins/org.talend.commons.runtime/pom_server.xml (hunk 1 above)
Comment (jiezhang-tlnd) [reply main/plugins/org.talend.commons.runtime/pom_server.xml:101]: OK

File: main/plugins/org.talend.utils/pom_minimal.xml (hunk 2 above)
Comment (jiezhang-tlnd) [reply main/plugins/org.talend.utils/pom_minimal.xml:44]: ok, thanks

**Pull request merged**